app/src/main/res/values/strings.xml

```diff
@@ -285,19 +285,29 @@ Please report any problems you encounter via Github.</string>
     <string name="category_debug">Debug</string>
     <string name="category_experimental">Experimental</string>
 
-    <string name="always_run_in_background">Always run in background</string>
+    <!-- Preference screen - Run conditions -->
+    <string name="always_run_in_background">to_be_deleted</string>
+    <string name="always_run_in_background_enabled">to_be_deleted</string>
+    <string name="always_run_in_background_disabled">to_be_deleted</string>
+    <string name="sync_only_charging">to_be_deleted</string>
+    <string name="sync_only_wifi">to_be_deleted</string>
+    <string name="sync_only_wifi_ssids">to_be_deleted</string>
 
-    <!-- Preference summary in case it is enabled -->
-    <string name="always_run_in_background_enabled">Syncthing always runs in the background, according to preferences below.</string>
+    <string name="run_conditions_title">Run Conditions</string>
+    <string name="run_conditions_summary">Use the following options to decide when the syncthing UI will be available and synchronization takes place.</string>
 
-    <!-- Preference summary in case it is disabled -->
-    <string name="always_run_in_background_disabled">Syncthing only runs when explicitly started, and can be stopped by menu button.</string>
+    <string name="run_on_mobile_data_title">Run on mobile data</string>
+    <string name="run_on_mobile_data_summary">Run when device is connected to the mobile data network. Attention: This can fastly consume your data plan if you sync a large amount of data.</string>
```

Comment:
Run when device is connected via the mobile data network. Warning: This can consume a lot of data from your mobile operator data plan if you sync large amounts of data.

Reply:
ok